test/plugin/base_test.rb

```diff
@@ -1623,12 +1629,10 @@ def setup_managed_vm_metadata_stubs
   def setup_container_metadata_stubs
     stub_metadata_request(
       'instance/attributes/',
-      "attribute1\nkube-env\nlast_attribute")
-    stub_metadata_request('instance/attributes/kube-env',
-                          "ENABLE_NODE_LOGGING: \"true\"\n"\
-                          'INSTANCE_PREFIX: '\
-                          "gke-#{CONTAINER_CLUSTER_NAME}-740fdafa\n"\
-                          'KUBE_BEARER_TOKEN: AoQiMuwkNP2BMT0S')
+      "attribute1\ncluster-name\nkube-env\nlast_attribute")
+    stub_metadata_request('instance/attributes/cluster-name',
+                          CONTAINER_CLUSTER_NAME)
+    stub_metadata_request_permission_denied('instance/attributes/kube-env')
```

Comment:
I think we'll want to have tests both with `kube-env` present and with it hidden. Perhaps add a `setup_container_metadata_stubs_no_kube_env` and a separate test that uses it?